scripts/pre_commit_linter.py

```diff
@@ -202,6 +202,16 @@
             'core/tests/performance_framework/perf_domain.py'),
         'excluded_dirs': ('scripts/',)
     },
+    {
+        'regexp': r'# pylint: disable=[A-Z][0-9]{4}',
```

Comment:
Change the space after the colon to something like `\s*`, because sometimes people put 0 or more spaces.

Reply:
@lilithxxx in future please make sure to reply to all review comments with (at least) "done". Thanks!